log/log.go

```diff
@@ -62,6 +62,7 @@ const auditTag = "[AUDIT]"
 // and also writes to stdout/stderr. It is safe for concurrent use.
 func New(log *syslog.Writer, stdoutLogLevel int, syslogLogLevel int) (Logger, error) {
 	if log == nil {
+		//lint:ignore ST1005 We like to keep punctuation at the end of the error line.
```

Comment:
I'd actually do this the other way: let's abide by the best practice. The only time we consciously flout the error capitalization and punctuation is when directly calling `cmd.Fail`, at which point we know we're exiting and no other message will precede ours.